run.py

```diff
@@ -96,17 +96,17 @@ def parse_suricata_version(buf):
         if m.group(1) is not None:
             major = int(m.group(1))
         else:
-            major = None
+            major = 0
```

Comment:
Minor comment about style: We could probably write all these blocks as `major = int(m.group(1)) if m.group(1) else 0`.

Reply:
That does clean it up a little. Thanks.